db/migrate/20230728145210_create_users.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+
```

Comment:
Good work implementing the user's table 💯 

- [ ] Kindly add to the migration (you can look to [how to add a column](https://guides.rubyonrails.org/active_record_migrations.html) in an existing table) the following column `t.integer :posts_counter` to match with the ERD diagram provided